app/models/whitehall_asset.rb

```diff
@@ -7,7 +7,9 @@ class WhitehallAsset < Asset
 
   validates :legacy_url_path,
     presence: true,
-    uniqueness: true,
+    uniqueness: {
+      conditions: -> { where(deleted_at: nil) }
+    },
```

Comment:
Very minor: Why does this need to be split onto multiple lines?

Reply:
We've used multiple lines for the `format` options so I did the same here.